### PR TITLE
Update zkEVM docs - Edit "User fees" doc

### PR DESCRIPTION
**Pull request by EmpieichO**
Update zkEVM docs - Edit "User fees" doc ... https://polygon.atlassian.net/browse/DOCS-79

### CODE REVIEW

File: docs/zkEVM/architecture/user-fees.md [hunk 1]
```diff
@@ -426,7 +417,18 @@ where $\texttt{TxNonZeroBytes}$ represents the count of non-zero bytes in a raw
 
 ### Computational costs
 
-Computational cost is calculated with the following formula:
+
+Costs associated with transaction execution is denoted by $\texttt{ExecutionCost}$, and it is measured in gas.
```
Comment (kmurphypolygon) [inline docs/zkEVM/architecture/user-fees.md:421]: do you mean gwei?

File: docs/zkEVM/architecture/user-fees.md [hunk 2]
```diff
@@ -426,7 +417,18 @@ where $\texttt{TxNonZeroBytes}$ represents the count of non-zero bytes in a raw
...
+
+In contrast to costs for data availability, calculating computational costs necessecitates transactions to be executed.
+
+So then,
+
+$$
+\texttt{GasUsed} = \texttt{DataCost} + \texttt{ExecutionCost}
+$$
+
+The total fees to be received by L2 are calculated with the following formula:
```
Comment (kmurphypolygon) [inline docs/zkEVM/architecture/user-fees.md]:
```suggestion
The total fees received by L2 are calculated with the following formula:
```

**Review by kmurphypolygon — APPROVED** [review]
LGTM, one suggestion and a question

File: docs/zkEVM/architecture/user-fees.md (hunk 1 above)
Comment (EmpieichO) [reply docs/zkEVM/architecture/user-fees.md:421]: Gas is measured in gas units ... and the fees are in Gwei.

Comment (kmurphypolygon) [reply docs/zkEVM/architecture/user-fees.md:421]: thanks for the clarification :)

**Pull request merged**